### PR TITLE
[13.0][ADD] fleet_vehicle_service_kanban: Kanban features in vehicle services.

### PR DESCRIPTION
**Pull request by mamcode**
This module adds the following features to vehicle services (fleet.vehicle.log.service):
* Kanban view.
* Features of mail (chatter) and acitivities, includes a new activity view.
* Configurable activity types. Menu under `Fleet / Configuration / Services Logs / Activity Types`.
* New fields: Responsible, Priority, Tags, Active, Stage.
* Configurable stages. Menu under `Fleet / Configuration / Services Logs / Vehicle Services Stages`.
* Configurable tags. Menu under `Fleet / Configuration / Services Logs / Vehicle Services Tags`.

screenshot-localhost_8099-2020 05 21-15_39_57

screenshot-localhost_8099-2020 05 21-15_40_34

screenshot-localhost_8099-2020 05 21-15_40_52

screenshot-localhost_8099-2020 05 21-15_43_20

screenshot-localhost_8099-2020 05 21-15_44_07

screenshot-localhost_8099-2020 05 21-15_45_48

### CODE REVIEW

File: fleet_vehicle_service_kanban/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Kanban Features for Vehicle Services",
+    "summary": "Add features of kanban to logs of vehicle services.",
+    "category": "Human Resources/Fleet",
+    "author": "Druidoo, Odoo Community Association (OCA)",
+    "maintainers": ["mamcode", "ivantodorovich"],
+    "development_status": "Stable",
+    "website": "https://www.druidoo.io/",
```
Comment (ivantodorovich) [inline fleet_vehicle_service_kanban/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/fleet",
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Comment by mymage** [comment]
Tested and goes well.
Only a doubt: isn't better to have a scale of priority instead an on/off?

**Review by mymage — APPROVED** [review]

**Comment by mamcode** [comment]
> Tested and goes well.
> Only a doubt: isn't better to have a scale of priority instead an on/off?

We try to make the improvements following the same usability features of project.task, but it may be interesting to have a priority scale for vehicle services, maybe in a next PR :)

**Pull request merged**